<p>
    There should be one single <code>Then</code> step per scenario. If you feel compelled to add more, it is usually a
    sign that you should split the scenario up in multiple scenarios or that some <code>Then</code> steps could be
    converted into more complete source code methods.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Scenario: Add a nice bike to my cart
  Given I am a customer
  When I go to the nice bike product page
  Then I add the nice bike to my cart
  And I should see the nice bike in my cart  # Noncompliant
</pre>

<h2>Compliant Solution</h2>
<pre>
Scenario: Add a nice bike to my cart
  Given I am a customer
  When I am on the nice bike product page
  Then I can add the nice bike to my cart
</pre>
